docs/spl-docs/regex-command.md

```diff
@@ -0,0 +1,72 @@
+# regex command
+
+## Description
+
+This command can be used to filter out results that either match or do not match the specified regular expression.
+
+## Syntax
+The required syntax is in **bold**.
+
+**regex**\
+(\<field\>=\<regex-expression\> | \<field\>!=\<regex-expression\> | **\<regex-expression\>**)
+
+
+### Required Arguments
+
+#### \<regex-expression\>
+
+**Syntax:** `"<string>"`\
+**Description:** \
+This argument must be an unanchored regular expression compatible with RE2 syntax and enclosed in double quotes.\
+If only `<regex-expression>` is present without any `<field>` argument, it will fetch all records where this `<regex-expression>` matches in any field.
```

Comment:
> This argument must be an unanchored regular expression ...

Is this true? I thought the expression is unanchored by default but you're still allowed to include anchors. But you should double check me

Reply:
addressed in the latest commit.